Assets/NoesisGUI/Xaml/Lobby.xaml

```diff
@@ -81,33 +81,84 @@
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                    </Grid.RowDefinitions>
+                                        <Grid x:Name="SERVER_CONTENT"  Grid.Row="0">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition/>
+                                            <ColumnDefinition/>
+                                            <ColumnDefinition/>
+                                            <ColumnDefinition/>
+                                            <ColumnDefinition/>
+                                        </Grid.ColumnDefinitions>
+                                        <TextBlock Foreground="#FFB2B2B2"  Grid.Column="0">Name</TextBlock>
+                                        <TextBlock Foreground="#FFB2B2B2"  Grid.Column="1">Map</TextBlock>
+                                        <TextBlock Foreground="#FFB2B2B2"  Grid.Column="2">Players</TextBlock>
+                                        <TextBlock Foreground="#FFB2B2B2"  Grid.Column="3">Ping</TextBlock>
+                                        <TextBlock Foreground="#FFB2B2B2"  Grid.Column="4">Status</TextBlock>
+                                    </Grid>
+                                    <ListBox x:Name="LOCAL_SERVER_LIST" Grid.Column="1" Grid.ColumnSpan="2" Grid.Row="1" Grid.RowSpan="10" Margin="5,0,10,0" Background="#FF171717">
+                                    </ListBox>
+                                </Grid>
                             </TabItem>
-                            <TabItem Header="Your Servers">
-                                <ListBox x:Name="SAVED_SERVER_LIST" Margin="5,0,10,0" Background="#FF171717">
+                            <TabItem x:Name="SAVED" Header="Saved Servers">
+                                <Grid Grid.Column="1" Grid.ColumnSpan="2">
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                        <RowDefinition/>
+                                    </Grid.RowDefinitions>
+                                    <Grid x:Name="SERVER_CONTENT_2"  Grid.Row="0">
```

Comment:
Recommended to have more  descriptive x:Name. SERVER_CONTENT_2 does not tell me what will go inside.

Reply:
Absolutely, will change on both Grids!